swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -233,12 +233,22 @@ Distributions available locally:
 
 ### Pull a specific distribution
 
-- The `ballerina dist pull <distribution>` command downloads a particular distribution and stores it in your local environment. It also sets the fetched distribution as the active distribution.
+The `ballerina dist pull <distribution>` command downloads a particular distribution and stores it in your local environment. It also sets the fetched distribution as the active distribution.
+
+#### For versions below jBallerina 1.2.5 (for update tool versions below 0.8.8):
```

Comment:
@praneesha  shall we interchange the order as 0.8.8 will be packed by default?

Reply:
Done.